benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -554,16 +558,16 @@ msgid "core.buttons.retry"
 msgstr "Intentar otra vez"
 
 msgid "enrollment.pages.success.title"
-msgstr "TODO: Success"
+msgstr "Éxito"
 
 msgid "enrollment.pages.success.headline"
-msgstr "¡Éxito! Su descuento está ahora vinculado a su tarjeta bancaria."
+msgstr "¡Felicidades! Su descuento ahora está vinculado a su tarjeta bancaria."
 
 #, python-format
 msgid "enrollment.pages.success.helplink%(link)s"
 msgstr ""
-"TODO: Thank you for using Cal-ITP Benefits! For more information check out "
-"our <a href=\"%(link)s\" rel=\"noopener noreferrer\">Help Page</a>."
+"Gracias por usar los Beneficios de Cal-ITP. Para más información, consulte "
+"nuestro our <a href=\"%(link)s\" rel=\"noopener noreferrer\">Help Page</a>."
```

Comment:
Link text here should be `Página de Ayuda`

Reply:
Done https://github.com/cal-itp/benefits/pull/1115/commits/dea38b1b7adb4497f6a81f4cfb6d8809d6a688d7